react/features/recording/components/Recording/AbstractStopRecordingDialog.ts

```diff
@@ -89,6 +90,9 @@ export default class AbstractStopRecordingDialog<P extends IProps>
             }
         }
 
+        // TODO: this should be an action in transcribing. -saghul
```

Comment:
do we plan this for a future pr?

Reply:
Yep, I plan on moving all transcriptions stuff to the transcriptions feature and leaving subtitles just about the visuals.

That will be a new PR yeah.